internal/receiver/databricksreceiver/json_types.go

```diff
@@ -0,0 +1,152 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package databricksreceiver
+
+// This file contains structs into which responses from the databricks API are
+// unmarshalled. There are two top-level types: jobsList and jobRun.
+
+// jobsList is a top level type
+type jobsList struct {
```

Comment:
Would you mind linking to the api docs for these response structures?

Reply:
Sure -- I've added a link to the API page which has pointers to the OpenAPI spec.